src/explore-education-statistics-admin/src/pages/publication/components/PublicationForm.tsx

```diff
@@ -201,140 +208,169 @@ const PublicationForm = ({
+                      methodologyId: orderBy(
+                        approvedMethodologies,
+                        methodology => methodology.title,
+                      )[0]?.id,
+                      externalMethodology: {
+                        title: '',
+                        url: '',
+                      },
+                    });
+                    break;
+                  case 'external':
+                    form.setValues({
+                      ...form.values,
+                      methodologyId: '',
+                    });
+                    break;
+                  default:
+                    form.setValues({
+                      ...form.values,
+                      methodologyId: '',
+                      externalMethodology: {
+                        title: '',
+                        url: '',
+                      },
+                    });
+                    break;
+                }
+              }}
             />
+            <FormFieldset
+              id="contact"
+              legend="Contact for this publication"
+              legendSize="m"
+              hint="They will be the main point of contact for data and methodology enquiries for this publication and its releases."
+            >
+              <FormFieldTextInput<FormValues>
+                name="teamName"
+                label="Team name"
+                className="govuk-!-width-one-half"
+              />
 
-            <FormFieldTextInput<FormValues>
-              name="teamEmail"
-              label="Team email address"
-              className="govuk-!-width-one-half"
-            />
+              <FormFieldTextInput<FormValues>
+                name="teamEmail"
+                label="Team email address"
+                className="govuk-!-width-one-half"
+              />
 
-            <FormFieldTextInput<FormValues>
-              name="contactName"
-              label="Contact name"
-              className="govuk-!-width-one-half"
-            />
+              <FormFieldTextInput<FormValues>
+                name="contactName"
+                label="Contact name"
+                className="govuk-!-width-one-half"
+              />
 
-            <FormFieldTextInput<FormValues>
-              name="contactTelNo"
-              label="Contact telephone number"
-              width={10}
-            />
-          </FormFieldset>
+              <FormFieldTextInput<FormValues>
+                name="contactTelNo"
+                label="Contact telephone number"
+                width={10}
+              />
+            </FormFieldset>
 
-          <ButtonGroup>
-            <Button type="submit">Save publication</Button>
-            {cancelButton}
-          </ButtonGroup>
-        </Form>
+            <ButtonGroup>
+              <Button
+                type="submit"
+                onClick={e => {
+                  e.preventDefault();
+                  if (confirmOnSubmit && form.isValid) {
```

Comment:
I think if confirmOnSubmit is true and form.isValid is false, then the form will submit?

Reply:
Yes, but then it'll show the validation errors and not save the changes. I figured it was better to show the validation errors before the confirm modal as it'd be a bit annoying otherwise.